app/views/boards/show.html.erb

```diff
@@ -46,6 +46,7 @@
   board: @board,
   cardsByType: @cards_by_type,
   creators: @board_creators,
+  current_creator: @current_creator,
```

Comment:
Adding this to react component probably expects that you will change a component on frontend side, at least a signature of the method. Right now it doesn't expect current_creator param at all. If merged we don't know what behaviour to expect. Have you checked locally, that board page still works properly?

Please also notice, that other params are passed in camelCase, and you are using snake case for you attribute. Since I believe you don't need this in component at all, that's not a big problem, but please be attentive to details.

Reply:
Got it, thanks.